server.go

```diff
@@ -0,0 +1,273 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/chromium/hstspreload"
+	"github.com/chromium/hstspreload/chromiumpreload"
+)
+
+func main() {
+	staticHandler := http.FileServer(http.Dir("files"))
+	http.Handle("/", staticHandler)
+	http.Handle("/static/", staticHandler)
+
+	http.HandleFunc("/robots.txt", http.NotFound)
+	http.HandleFunc("/favicon.ico", http.NotFound)
+
+	http.HandleFunc("/checkdomain/", checkdomain)
+	http.HandleFunc("/status/", status)
+
+	http.HandleFunc("/submit/", submit)
+	http.HandleFunc("/pending", pending)
+	http.HandleFunc("/update", update)
```

Comment:
Also I gather that main() is only used when running locally and appspot handles this automatically. Is that right?